src/main/java/mate/academy/internetshop/util/HashUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.internetshop.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.apache.log4j.Logger;
+
+public class HashUtil {
+    private static final org.apache.log4j.Logger LOGGER = Logger.getLogger(HashUtil.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(HashUtil.class);
```
повний шлях тут вказувати не обов'язково

Reply:
done